.github/workflows/main.yml

```diff
@@ -4,9 +4,9 @@ name: CI
 
 on:
   push:
-    branches: [ main, 1.0.1-release, 1.0.0-release ]
+    branches: [ main, 1.0.2-release ]
```

Comment:
let's keep `1.0.1-release` so that, if we need to release it with next Kotlin release, it is tested.